android/sdk/src/main/java/com/taobao/weex/ui/WXRenderManager.java

```diff
@@ -144,6 +165,15 @@ public void postGraphicAction(final String instanceId, final BasicGraphicAction
       return;
     }
 
+    if (mCurrentBatchInstanceId != null && instanceId != null && !mCurrentBatchInstanceId.equals(instanceId) && mBatchActions.size() > 0) {
```

Comment:
Will your change makes `public void postGraphicAction(final String instanceId, final BasicGraphicAction action)` behaviour unexpectedly if there are two weex instance as you only use one field called `mCurrentBatchInstanceId`?

Reply:
If more than two pages exist and mCurrentBatchInstanceId not matches new instanceId, we will post all stashed actions at once.

That will cause losing efficacy of batch action, but it is acceptable because it's not serious problem.